python/pyspark/sql/readwriter.py

```diff
@@ -1257,14 +1214,13 @@ def parquet(self, path, mode=None, partitionBy=None, compression=None):
             * ``ignore``: Silently ignore this operation if data already exists.
             * ``error`` or ``errorifexists`` (default case): Throw an exception if data already \
                 exists.
-        partitionBy : str or list, optional
```

Comment:
I think `partitionBy` is not parquet specific option. We'll have to recover here and remove it in the docs.